python/tests/test_aggregates.py

```diff
@@ -0,0 +1,335 @@
+import unittest
+from feldera import SQLContext
+from tests import TEST_CLIENT
+
+class TestAggregates(unittest.TestCase):
```

Comment:
let's structure this use classes.
There should be a base class TestAggregatesBase, and all tests derive from this class.
The class has fields for the table, the view, the input data, and the expected results
I think the table is the same in all tests, so it should be part of the base class too
The code to run the test is entirely in the base class:
```
        out = sql.listen("avg_view_value")
        sql.start()
        sql.input_json("avg0_value", data)
        sql.wait_for_completion(True)
        out_data = out.to_dict()
        for datum in expected_data:
            datum.update({"insert_delete": 1})
        assert expected_data == out_data
```

Reply:
So the code here will become one base class and 4 derived classes, for AVG, AVG DISTINCT, AVG WHERE, and AVG GROUPBY